src/components/studies/StudyBuilder.tsx

```diff
@@ -396,17 +451,33 @@ const StudyBuilder: FunctionComponent<StudyBuilderProps> = ({
   }
 
   const getBannerType = () => {
+    if (builderInfo?.study?.phase === 'in_flight') return 'live'
```

Comment:
I think a switch statement might be appropriate here w/ 'completed' and 'analysis' be the default ?